web/testing/issue103/index.html

```diff
@@ -40,14 +32,14 @@
       The toolbar UI is best for any page designed to support keyboards for 
       a large number of languages.
     -->
-    <script src="../../source/kmwuitoggle.js"></script>                                                                                           
+    <script src="../../release/unminified/web/kmwuitoggle.js"></script>
  
     <!-- Initialization: set paths to keyboards, resources and fonts as required -->
     <script>
-      var kmw=window.tavultesoft.keymanweb;
+      var kmw=window.keyman;
       kmw.init({
         attachType:'auto',
-        resources:'resources'
+        resources:''
         });
     </script> 
     
```

Comment:
l45, Don't you want to use commonHeader.js instead of the old samplehdr.js?

Reply:
Maybe you are intending to stick with samplehdr.js for issue103. If that's the case, line 50 can change from `language:` to `languages:`...

Reply:
I do not want to do so, no.  Note that I'm using neither for this issue's pages, as the script has been modified - some keyboards have been requested multiple times, as handling this scenario correctly is the purpose of the test.

I suppose it may be possible to refactor into use of commonHeader.js with the duplicate requests in this issue's custom script file, but I feel that might result in an unclear picture for the issue.